pangeo_forge_orchestrator/routers/github_app.py

```diff
@@ -795,21 +941,36 @@ async def synchronize(
             if ("status" in p) and p["status"] == "failed":
                 tracelines = p["exc_info"].splitlines()
                 logger.debug(f"Synchronize errored with:\n {tracelines}")
-                update_request = dict(
+
+                summary = dedent(
+                    f"""\
+<details>
+<summary>Traceback: Click to expand</summary>
+
+```python
+{tracelines[-1]}
```

Comment:
@cisaacstern, i'm curious... is there any particular reason for not exposing the full traceback here?

Reply:
@andersy005, I think this was because the traces from `pangeo-forge-runner` can be pretty long, with a lot of boilerplate piled on top of the lines we care about at the end. So for the user who will be seeing this on GitHub, I guess I figured it would be clearer to just show the last line, which might be something like:

```
FileNotFoundError: No file named meta.yaml
```
if they don't have a `meta.yaml`. Or:

```
NameError: such and such is not defined
```
if they use an undefined name.

Reply:
Shoot I wrote a long response to this that I seem to have accidentally deleted 🤦 .

Anyway, the answer is that the traces can be very long, and for the average user on GitHub I think would be potentially confusing.

Typically, the error type (captured in the last line of the trace) is all we need to know to figure out what's going on here. It will typically be `FileNotFoundError` if the required files (i.e. `meta.yaml`) are missing, or possibly `NameError` if the recipe uses an undefined name.

Open to any improved presentation here, though I would be skeptical that displaying the full trace improves clarity here. Perhaps some custom markdown could be an improvement, with the message dependent on the type of error.